anyway_etl/waze/import_to_db.py

```diff
@@ -0,0 +1,33 @@
+import os
+import dataflows as DF
+
+from anyway_etl.waze.config import FIELDS, TYPES_MAPPING
+from anyway_etl.db import session_decorator
+
+
+def _get_saved_data(datapackage_filename):
+    return DF.Flow(DF.load(datapackage_filename)).results()[0][0]
+
+
+@session_decorator
+def _import_data(session, field: str, data: list):
+    data_type = TYPES_MAPPING[field]
+    session.bulk_insert_mappings(data_type, data)
```

Comment:
Me and @atalyaalon have discussed about it and have decided that we'll insert new data on every iteration, in the beginning.
Managing duplicated rows is highly recommended.

Reply:
thanks, I can't approve it as it is without some explanation about this decision

other then that, it looks fine, please re-request a review if you are finished making your fixes

Reply:
@atalyaalon these are the generated files (generated now from my PC):
[jams.csv](https://github.com/hasadna/anyway-etl/files/7112095/jams.csv)
[alerts.csv](https://github.com/hasadna/anyway-etl/files/7112096/alerts.csv)

Each item has a created_at field which should be used to prevent duplicate items

The files contain a lot of old data, so at the moment, merging this PR will cause hundreds of lines to be added to the DB each day.

I don't see any possible reason to do it this way.. The data can be seen right now, there is no added value to have all this duplicated data in DB.

I can't approve the PR as-is without handling of duplicated data

Reply:
@carmelp16 is there a way to keep track on the time range in which a specific accident was present in the data? If so we can eliminate the duplicates and make sure we save this time range (say start date and end date in our DB)

Reply:
@OriHoch I am not sure I have understood how the `created_at` field would help in regard to updating the DB.
Could you please elaborate on that?

Reply:
@carmelp16 I didn't fully research and spec this but I suppose you can iterate over the rows in the datapackage, and before inserting a new row, check if an existing row already exists in DB by searching for rows with the same created_at value as well as some other fields like city/street